src/Compilers/VisualBasic/Test/Semantic/Semantics/IteratorTests.vb

```diff
@@ -1023,6 +1091,51 @@ End Class
             CompileAndVerify(compilation, expectedOutput:="123")
         End Sub
 
+        <Fact>
+        Public Sub IteratorProperty_Metadata()
+            Dim sourceComp = CreateCompilation(
+<compilation>
+    <file name="a.vb"><![CDATA[
+Imports System.Collections.Generic
+
+Class C
+    Iterator ReadOnly Property P As IEnumerable(Of Integer)
+        Get
+            Yield 1
+        End Get
+    End Property
+End Class
+]]></file>
+</compilation>).VerifyDiagnostics()
+
+            Dim userComp = CreateCompilation("", references:={sourceComp.EmitToImageReference()}).VerifyDiagnostics()
+            Dim cMetadataType = Assert.IsAssignableFrom(Of PENamedTypeSymbol)(userComp.GetTypeByMetadataName("C"))
+
+            Dim [property] = cMetadataType.GetProperty("P")
+            Assert.False([property].GetMethod.IsIterator)
+        End Sub
+
+        <Fact>
+        Public Sub Property_NotIterator()
+            Dim compilation = CreateCompilation(
+<compilation>
+    <file name="a.vb"><![CDATA[
+Imports System.Collections.Generic
+
+Class C
+    ReadOnly Property P As IEnumerable(Of Integer)
+        Get
+            Return System.Array.Empty(Of Integer)()
+        End Get
+    End Property
+End Class
+]]></file>
+</compilation>).VerifyDiagnostics()
+
+            Dim [property] = compilation.GetMember(Of PropertySymbol)("C.P")
+            Assert.False([property].GetMethod.IsIterator)
```

Comment:
>Assert.False([property].GetMethod.IsIterator) [](http://example.com/codeflow?start=12&length=45)

Is there a positive test? Once you submit a proposal for the new public API, please make sure to include proposed handling of properties.

Reply:
I thought you have a specialized review software and you see the full picture, not exclusively the diffs... The test you are looking for has already existed before this PR (excluding the fact, that it wasn't testing interface property, but that is another feedback item from you), it is 2 tests above this on (`WritableIteratorProperty`). I added `SimpleIteratorProperty` test case to verify get-only property if you meant that and I misunderstood you